tensorflow_quantum/core/serialize/serializer.py

```diff
@@ -154,6 +154,34 @@ def _optional_control_promote(gate, qubits_message, values_message):
     return DelayedAssignmentGate(gate, qbs, vals)
 
 
+def _depolarize_channel_serializer():
+    """Make standard serializer for depolarization channel."""
+
+    args = [
+        # cirq channels can't contain symbols.
+        cirq.google.SerializingArg(serialized_name="p",
+                                   serialized_type=float,
+                                   op_getter=lambda x: x.gate.p)
+    ]
+    return cirq.google.GateOpSerializer(gate_type=cirq.DepolarizingChannel,
+                                        serialized_gate_id="DP",
+                                        args=args,
+                                        can_serialize_predicate=_CONSTANT_TRUE)
```

Comment:
what happens to this line...? it seems over 80 characters, but not formatted without error..?

Reply:
It doesn't show up locally as being over 80. In the webui it looks a little weird for me too....

Reply:
Yes, now I checked that it's not over 80 characters, floating due to HTML... (we have `+` at the front of the line, so it warped.) LGTM.